### DISCUSSION TITLE
What is the best way to share vue components with tailwindcss between projects?

### DISCUSSION THREAD

**Discussion by romanslonov** [tailwindlabs/tailwindcss#3405] [Help] [answered]
Hey everyone, 

I am using vue and tailwindcss for a while on several project and really love it. Now, I need to share some components across few projects and created an npm package with that components. Example of component: 

```js
<template>
  <button class="v-button">
      <slot />
  </button>
</template>

<script>
export default {}
</script>

<style lang="postcss">
.v-button {
  @apply bg-blue-500 text-white rounded h-12 px-4;
</style>
```

The one obviously disadvantage is sharing tailwind config (?).

So maybe there is a better approach to accomplish that goal? Examples? 

Would really appreciate to any help!

**Comment by hylickipiotr** [comment]
@romanslonov Did you find solution to this problem?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Perhaps you could have a preset that gets imported in each of your projects:

https://tailwindcss.com/docs/presets

This is a nice way to share the same config customisation across multiple projects.

You can even have multiple preset "partials", and then use a combination of whatever makes sense for each project:

https://tailwindcss.com/docs/presets#extending-multiple-presets

Hope that helps you getting started 👍

**Reply by hylickipiotr** [reply]
@simonswiss  Thank you for your help 😄
